data-structures-queues/is-empty.js

```diff
@@ -0,0 +1,6 @@
+export default function isEmpty(queue) {
+  if (queue.peek() === undefined) {
+    return true;
+  }
+  return false;
```

Comment:
It's best to avoid structures like `if (x) return true; else return false;` since just `return x;` does the trick.